data/transaction/apiTransactionResult.go

```diff
@@ -11,6 +11,9 @@ import (
 type ApiTransactionResult struct {
 	Tx                                data.TransactionHandler   `json:"-"`
 	Type                              string                    `json:"type"`
+	ProcessingTypeOnSource            string                    `json:"processingTypeOnSource,omitempty"`
+	ProcessingTypeOnDestination       string                    `json:"processingTypeOnDestination,omitempty"`
+	HashBytes                         []byte                    `json:"-"`
```

Comment:
maybe you can move `HashBytes` field after `Hash`  field in order to keep field with the same type togherder

Reply:
Fixed.